templates/beta-post.adoc

```diff
@@ -0,0 +1,186 @@
+----
+<dependency>
+    <groupId>io.openliberty.beta</groupId>
+    <artifactId>openliberty-jakartaee9</artifactId>
+    <version>20.0.0.RELEASE_VERSION-beta</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-jakartaee9', version: '[20.0.0.RELEASE_VERSION-beta,)'
+}
+----
+// // // // // // // //
+// Above:
+// Replace RELEASE_VERSION with the version number of Open Liberty
+// // // // // // // //
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+Enable the Jakarta EE 9 beta features in your app's `server.xml`. You can enable the individual features you want (but remember if you enable the `jsp-3.0` feature, the `servlet-5.0` and `el-4.0` features are automatically enabled for you) or you can just add the Jakarta EE 9 convenience feature to enable all of the Jakarta EE 9 beta features at once:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>jakartaee-9.0</feature>
+  </featureManager>
+----
+
+Or you can add the Web Profile convenience feature to enable all of the Jakarta EE 9 Web Profile beta features at once:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>webProfile-9.0</feature>
+  </featureManager>
+----
+
+=== HEADING
+
+// // // // // // // //
+// Remove if nothing else to say, or provide an update on aspect of the Jakarta EE 9 beta e.g- TCK
+// // // // // // // //
+
+[#allbeta]
+== All Beta Features package
+
+As well as the Jakarta EE 9 features listed above, the new All Beta Features package contains our new FEATURES features which have been available in our link:/downloads/#development_builds[nightly builds] for a few weeks but are now easy to install from Maven Central too.
+// // // // // // // //
+// Above:
+// Change FEATURES to a feature that is included in this release     // version.
+// // // // // // // //
+
+[#SUB_TAG_1]
+=== SUB_FEATURE_TITLE
+
+// // // // // // // //
+// FURTHER EXPLENATION OF THE FEATURE/CHANGE ABOVE
+// // // // // // // //
+
+
+
+[source, java]
+----
+// // // // // // // //
+// EXAMPLE CODE
+// // // // // // // //
+----
+
+
+=== Try it now section for the feature 
+
+// // // // // // // //
+// how to try it now
+// // // // // // // //
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>20.0.0.RELEASE_VERSION-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[20.0.0.RELEASE_VERSION-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+To enable the new beta features in your app, add them to your `server.xml`:
+
+[source, xml]
+----
+
```

Comment:
Missing an overall Try it now section for the All Beta Features package. I think to avoid repetitive headings, maybe remove the "Try it now" sub-headings (`===` headings) and instead just have a lead-in sentence that says something like "To try it out, add the following features to your `server.xml`:" (if that makes sense in context).